components/Card/ImageCard/CardImage.js

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image';
+import React from 'react';
+
+export default function CardImage({
```

Comment:
same as here, the CardImage can be rename to only Image because it is in the card folder